pkg/diff/testdata/smd-deploy-live.yaml

```diff
@@ -120,7 +125,10 @@ spec:
           ports:
             - containerPort: 80
               protocol: TCP
-          resources: {}
+          resources:
+            requests:
+              memory: 512Mi
+              cpu: 500m
```

Comment:
Rather than change the existing test, should we create a new test and leave the old one in place?

Reply:
Alright, I'll create a 2nd deployment manifests files.